application/src/main/java/org/thingsboard/server/actors/tenant/TenantActor.java

```diff
@@ -221,6 +227,10 @@ private void onToDeviceActorMsg(DeviceAwareMsg msg, boolean priority) {
         if (!isCore) {
             log.warn("RECEIVED INVALID MESSAGE: {}", msg);
         }
+        if (deletedDevices.contains(msg.getDeviceId())) {
+            log.warn("RECEIVED MESSAGE FOR DELETED DEVICE: {}", msg);
```

Comment:
Please replace with debug level. Avoid warn logs for such a minor issue in the future.